site/using-weave/dynamically-attach-containers.md

```diff
@@ -37,6 +37,13 @@ specify `--rewrite-hosts` when running `weave attach`:
 
     host1$ weave attach --rewrite-hosts c1
 
+Weave Net will set the bridge side of the `veth` pair to enable hairpin mode when attaching a container.
```

Comment:
For extra clarity, here is the line where `weave attach` will call down to `weaveutil`: https://github.com/deitch/weave/blob/7a83fd829b9b65cc3c358cb8fd801fcf024f06bb/weave#L1507

it does not pass any arguments straight from the command-line.